### DISCUSSION TITLE
Restore Windows backup to Linux container instance

### DISCUSSION THREAD

**Discussion by PeterHageus** [datalust/seq-tickets#2107] [Help] [answered]
Hi. Is it possible to restore a config backup taken on a windows instance to a container hosted instance? 

Getting this message: 

running init script init.sh...
A valid .seqbac filename must be specified.

Versions should match.

Is it an alternative to go through a SQL config otherwise?

**Comment by PeterHageus** [comment, marked as answer]
Answering my own question for future reference: The relative path in the example did not work (../Backups/xxx.seqbac). Absolute path did (/data/Backups/xxx.seqbac)

**Reply by nblumhardt** [reply]
I've fixed the example, thanks for the heads-up :+1:
